dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/EnrollmentOperationParamsMapper.java

```diff
@@ -63,14 +65,18 @@ class EnrollmentOperationParamsMapper {
   public EnrollmentQueryParams map(
       @Nonnull EnrollmentOperationParams operationParams, @Nonnull UserDetails user)
       throws BadRequestException, ForbiddenException {
-    Program program = paramsValidator.validateTrackerProgram(operationParams.getProgramUid(), user);
+    Program program =
+        paramsValidator.validateTrackerProgram(
```

Comment:
Not really because these are optional parameters.
This null handling will disappear when the bullet point `Harmonize UID in OperationsParamsValidator` is tackled